### PR TITLE
Add current language from ISteamApps

### PR DESCRIPTION
**Pull request by GeorgeA93**
Adds the GetCurrentGameLanguage from the ISteamApps interface. This will
return a string of the users current language as per the steam docs.
Super useful for localisation purposes!

I have tested this locally on linux64 using the example code in the docs.

### CODE REVIEW

**Comment by GeorgeA93** [comment]
@yancouto when you have some time i'd love a review

File: src/apps.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+void add_apps(lua_State *L) {
+    lua_createtable(L, 0, 1);
+    add_func(L, "getCurrentGameLanguage", luasteam_getCurrentGameLanguage);
+    lua_setfield(L, -2, "apps");
+}
+
+void init_apps(lua_State *L) {}
+
+void shutdown_apps(lua_State *L) {}
```
Comment (yancouto) [inline src/apps.cpp:23]: Can you please add init and shutdown calls inside `luasteam_init`? I know we don't do anything now, but might do in the future.

File: docs/apps.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Function Reference
+------------------
+
+.. function:: apps.getCurrentGameLanguage()
+
+    :returns: (`string`) The language that the user has set.
+    :SteamWorks: `GetCurrentGameLanguage <https://partner.steamgames.com/doc/api/ISteamApps#GetCurrentGameLanguage>`_
+
+	Gets the current language that the user has set. This falls back to the Steam UI language if the user hasn't explicitly picked a language for the title.
```
Comment (yancouto) [inline docs/apps.rst:19]: Point to the full list of languages like in the steam docs.

**Review by yancouto — APPROVED** [review]
Awesome!! Looks really good. Just address the comments I made and I'll merge it.

**Comment by GeorgeA93** [comment]
@yancouto Done

**Comment by yancouto** [comment]
The latest changes broke the build. Take a look at Travis [here](https://travis-ci.org/github/uspgamedev/luasteam/jobs/766607684). Can you fix it?

No hurry, just need to fix this before we do another release.

**Comment by GeorgeA93** [comment]
@yancouto sure, think I missed an import...!

**Pull request merged**